src/GeneralStateTestsFiller/stCreateTest/CREATE_ContractRETURNBigOffsetFiller.json

```diff
@@ -0,0 +1,44 @@
+{
+    "CREATE_ContractRETURNBigOffset" : {
+        "env" : {
+            "currentCoinbase" : "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x020000",
+            "currentGasLimit" : "89128960",
+            "currentNumber" : "1",
+            "currentTimestamp" : "1000",
+            "previousHash" : "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "pre" :
+        {
+            "a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "100000000000",
+                "code" : "",
+                "nonce" : "0",
+                "storage" : {
+                }
+            }
+        },
+        "transaction" : {
+            "//data" : "{ (RETURN 0x074ac2 0x0d15bc) }",
+            "//data" : "{ (RETURN 0x074ac2 0x010000) }",
+            "//data" : "{ (RETURN 0x074ac2 0x051eb9) }",
+            "//data" : "{ (RETURN 0x074ac2 0x051eb8) }",
```

Comment:
The ordering of these lines does not correspond to the ordering in the actual `data` field.